src/json-patch-duplex.js

```diff
@@ -612,3 +609,6 @@ if (typeof exports !== "undefined") {
     exports.validator = jsonpatch.validator;
     exports.JsonPatchError = jsonpatch.JsonPatchError;
 }
+Object.defineProperty(exports, "__esModule", { value: true });
```

Comment:
Shouldn't we move those lines inside `if (typeof exports !== "undefined") {`? -  not to break support for envs w/o exports